ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -526,6 +527,22 @@ impl Module {
     ) -> Result<GuardInfoIdx, CompilationError> {
         GuardInfoIdx::try_from(self.guard_info.len()).inspect(|_| self.guard_info.push(info))
     }
+
+    pub(crate) fn loop_start_vars(&self) -> &Vec<Operand> {
```

Comment:
We shouldn't return `&Vec<T>` except in very unusual circumstances: `&[T]` is the standard Rust idiom.

Reply:
[9fa086d](https://github.com/ykjit/yk/pull/1381/commits/9fa086d73cfef632c2105203f5d7017b9c6eb7a7)